controllers/app.controller.js

```diff
@@ -0,0 +1,10 @@
+const {fetchCategories} = require("../models/app.model")
+
+const getCategory = (request, response, next) => {
+    fetchCategories().then((categories) => {
+        response.status(200).send({categories})
+    
+    })
+}
```

Comment:
We should have a .catch block here, sorry I should have said before. You can invoke next within it.